packages/bpk-component-page-indicator/index.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import BpkPageIndicator, {
+  type Props as BpkPageIndicatorProps,
+} from './src/BpkPageIndicator';
+
+export type { BpkPageIndicatorProps };
```

Comment:
Why export it?

Reply:
I found that props are exported in many other components even though they don't use them in the backpack project.🤔 So I exported them as well.